storybook/src/example-pages/Patronen/modal-return-focus.ts

```diff
@@ -1,7 +1,32 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
 import { html } from "lit-html";
+
 import { examplePageFactory } from "../../example-page-factory";
 
-examplePageFactory("Patronen", "Modal return focus", ({ buttonTemplate, modalTemplate }) => {
+examplePageFactory("Patronen", "Modal return focus", ({ buttonTemplate }) => {
+  const createModal = () => {
+    const modal = document.createElement("dso-modal");
+
+    modal.setAttribute("modal-title", "Modal");
+
+    const body = document.createElement("div");
+    body.setAttribute("slot", "body");
+    body.innerText = "test";
+
+    const footer = document.createElement("div");
+    footer.setAttribute("slot", "footer");
+    footer.innerHTML = "<button type='button' class='dso-primary'>Bevestigen</button>";
+
+    modal.appendChild(body);
+    modal.appendChild(footer);
+
+    modal.addEventListener("dsoClose", () => document.querySelector("main")?.removeChild(modal));
+
+    return modal;
```

Comment:
Waarom kunnen we dit niet in lit templating doen?

Reply:
Dat kan inderdaad wel. Was de render functie helemaal vergeten.

Reply:
👍